.rhcicd/clowdapp-backend.yaml

```diff
@@ -153,6 +153,7 @@ objects:
           - name: notifications-db-cleaner-volume
             configMap:
               name: notifications-db-cleaner-config
+        volumeMounts:
```

Comment:
what is this good for?

Reply:
where to mount it in the container, the configmap above is what to mount